changelog/bugfixes/2022-08-22-eks-support-for-versions-above-1.21.md

```diff
@@ -0,0 +1 @@
+- Added EKS support for version 1.22 and 1.23. Fixed `/usr/share/oem/eks/download-kubelet.sh` to include download paths for these versions. ([coreos-overlay#2110](https://github.com/flatcar-linux/coreos-overlay/pull/2110))
```

Comment:
Looks great, I think we can even remove the technical implementation, provide your initial issue and mention the vendor.
```suggestion
- AWS: added EKS support for version 1.22 and 1.23. ([coreos-overlay#2110](https://github.com/flatcar-linux/coreos-overlay/pull/2110), [Flatcar#829](https://github.com/flatcar-linux/Flatcar/issues/829))
```